duckduckgo/Android
# Improve multiprocess init

Task/Issue URL: https://app.asana.com/0/488551667048375/1203012803428746/f

### Description
See [task](https://app.asana.com/0/488551667048375/1203012803428746/f) for way more info.

TL;DR clean up (hopefully improve) the way we initialize things in the different processes we have

### Steps to test this PR
Smoke tests for browser and AppTP.

Check the firebutton and the logs to ensure it executes (or doesn't execute) what it should (or should not)
